docs/backup.md

```diff
@@ -145,6 +145,50 @@ Finally, the Job updates MySQLCluster status field with the following informatio
 - The maximum usage of the working directory
 - Warnings, if any
 
+When executing an incremental backup, the backup source must be a pod whose server_uuid has not changed since the last backup.
+If the server_uuid has changed, the pod may be missing some of the binlogs generated since the last backup.
+
+The following is how to choose a pod to be the backup source.
+
+```mermaid
+flowchart TD
+A{"first time?"}
+A -->|"yes"| B
+A -->|"no"| C["x ← Get the indexes of the pod whose server_uuid has not changed"] --> D
+
+B{Are replicas available?}
+B -->|"yes"| B1["return\nreplicaIdx\nskipBackupBinlog=true"]
+style B1 fill:#c1ffff
+B -->|"no"| B2["return\nprimaryIdx\nskipBackupBinlog=true"]
+style B2 fill:#ffffc1
+
+D{"Is x empty?"}
+D -->|"yes"| E["add warning to bm.warnings"] --> F
+style E fill:#ffc1c1
+D -->|"no"| G
```

Comment:
In my opinion, it would be easier to read if the paths for the selections of the primary idx and the replica idx were separated.

For example, it look like this.

```mermaid
flowchart TD
D{"Is x empty?"}
D -->|"yes"| E["add warning to bm.warnings"] --> F
style E fill:#ffc1c1
D -->|"no"| G
F{"Are replicas available?"}
F -->|"yes"| F1["return\nreplicaIdx\nskipBackupBinlog=true"]
style F1 fill:#ffc1c1
F -->|"no"| F2["return\nprimaryIdx\nskipBackupBinlog=true"]
style F2 fill:#ffc1c1
G{"Are there replica indexes in x?"}
G -->|"yes"| G1["y ← The set of indexes of replica pod of x"]--> H
G -->|"no"| G2["return\nprimaryIdx\nskipBackupBinlog=false"]
style G2 fill:#ffffc1
H{"Is lastIndex included in y?"}
H --> |"yes"| H1["return\nlastIdx\nskipBackupBinlog=false"]
style H1 fill:#c1ffff
H --> |"no"| H2["return\nreplicaIdx\nskipBackupBinlog=false"]
style H2 fill:#c1ffff
```

Reply:
9ce979f